cms/envs/common.py

```diff
@@ -738,23 +769,9 @@
 # The name of the require.js script used by your project, relative to REQUIRE_BASE_URL.
 REQUIRE_JS = "js/vendor/requiresjs/require.js"
 
-# A dictionary of standalone modules to build with almond.js.
```

Comment:
Reviewers: I removed settings that were just re-setting defaults, as they added unnecessarily to the complexity.